regression/Makefile

```diff
@@ -82,18 +82,27 @@ $(INITRAMFS)/benchmark:
 $(INITRAMFS_EMPTY_DIRS):
 	@mkdir -p $@
 
+.PHONY: $(INITRAMFS)/opt/syscall_test
 $(INITRAMFS)/opt/syscall_test:
 	@make --no-print-directory -C syscall_test
 
+.PHONY: $(INITRAMFS_IMAGE)
+$(INITRAMFS_IMAGE): initramfs_filelist
+	@if ! cmp -s $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).old ; then \
+		echo "Generating the initramfs image..."; \
+		cp $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).old; \
+		(cd $(INITRAMFS); find . | cpio -o -H newc | gzip) > $@; \
```

Comment:
Why not just cat the file `initramfs_filelist` and call cpio to generate the image?

Reply:
`$(INITRAMFS_FILELIST)` contains the modification time to track when these files are updated in initramfs. The modification time must be removed before passing its contents to `cpio`.

I changed it to `cut -d "" -f 2-` ("2-" to allow spaces in filenames, although we should avoid spaces there), but I'm not sure if that's better or not. Both are acceptable to me.

Example:
```
> cat regression/build/initramfs.filelist | head -10
1709231337.5710777910 .
1709231334.8777257870 ./etc
1709231334.8777257870 ./etc/passwd
1709231334.8777257870 ./etc/group
1709231336.4677368110 ./lib
1709231336.4944036620 ./lib/x86_64-linux-gnu
1709231336.4744035230 ./lib/x86_64-linux-gnu/libc.so.6
1709231336.4777368790 ./lib/x86_64-linux-gnu/libstdc++.so.6
1709231336.4810702360 ./lib/x86_64-linux-gnu/libm.so.6
1709231336.4810702360 ./lib/x86_64-linux-gnu/libgcc_s.so.1
```
```
> cat regression/build/initramfs.filelist | cut -d " " -f 2- | head -10
.
./etc
./etc/passwd
./etc/group
./lib
./lib/x86_64-linux-gnu
./lib/x86_64-linux-gnu/libc.so.6
./lib/x86_64-linux-gnu/libstdc++.so.6
./lib/x86_64-linux-gnu/libm.so.6
```